lib/betamocks/middleware.rb

```diff
@@ -16,12 +16,13 @@ def call(env)
         @response_cache = Betamocks::ResponseCache.new(env: env, config: @endpoint_config)
         response = @response_cache.load_response
         return response if response
+        return @response_cache.load_response('default.yml') if Betamocks.configuration.mode == Betamocks::Configuration::PLAYBACK
```

Comment:
could make a helper method to check the mode `if Betamocks.configuration.playback?` or maybe we only need one a `recording?` method and switch it to unless `unless Betamocks.configuration.recording?`